apps/web/src/components/AppSidebar.tsx

```
@@ -351,6 +351,35 @@ const AppSidebar = () => {
+            class="rounded p-1.5 text-muted-foreground transition-colors hover:bg-accent hover:text-foreground"
+            title="Settings"
+            aria-label="Settings"
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              class="h-4 w-4"
+              fill="none"
+              viewBox="0 0 24 24"
+              stroke="currentColor"
+              stroke-width="2"
+            >
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)